lib/internal/quic/common.js

```diff
@@ -0,0 +1,614 @@
+'use strict';
+
+const {
+  FunctionPrototypeCall,
+  ObjectDefineProperties,
+  PromisePrototypeThen,
+  PromisePrototypeFinally,
+  PromiseReject,
+  PromiseResolve,
+  ReflectConstruct,
+  Symbol,
+  SymbolIterator,
+  SymbolAsyncIterator,
+  Uint8Array,
+} = primordials;
+
+const {
+  codes: {
+    ERR_ILLEGAL_CONSTRUCTOR,
+    ERR_INVALID_ARG_TYPE,
+    ERR_INVALID_THIS,
+  },
+} = require('internal/errors');
+
+const {
+  Event,
+} = require('internal/event_target');
+
+const {
+  ArrayBufferViewSource,
+  BlobSource,
+  StreamSource,
+  StreamBaseSource,
+} = internalBinding('quic');
+
+const {
+  createDeferredPromise,
+  customInspectSymbol: kInspect,
+} = require('internal/util');
+
+const {
+  isAnyArrayBuffer,
+  isArrayBufferView,
+  isPromise,
+  isGeneratorObject,
+} = require('util/types');
+
+const {
+  Readable,
+  Writable,
+  pipeline,
+} = require('stream');
+
+const {
+  Buffer,
+} = require('buffer');
+
+const {
+  isBlob,
+  kHandle: kBlobHandle,
+} = require('internal/blob');
+
+const {
+  inspect,
+  TextEncoder,
+} = require('util');
+
+const {
+  isReadableStream,
+} = require('internal/webstreams/readablestream');
+
+const {
+  WritableStream,
+} = require('internal/webstreams/writablestream');
+
+const {
+  kEnumerableProperty,
+} = require('internal/webstreams/util');
+
+const {
+  getPromiseDetails,
+  kPending,
+} = internalBinding('util');
+
+const {
+  isFileHandle,
+  kHandle: kFileHandle,
+} = require('internal/fs/promises');
+
+const {
+  symbols: {
+    owner_symbol,
+  },
+} = require('internal/async_hooks');
```

Comment:
Is there a reason for getting `owner_symbol` by requiring `internal/async_hooks` rather than using `internalBinding('symbols')`? Requiring `internal/async_hooks` has side effects that might be best avoided when not needed.

```suggestion
const { owner_symbol } = internalBinding('symbols');
```